README.md

```diff
@@ -1,12 +1,16 @@
 # TOPMed Analysis Pipeline — WDL Version
 
 [![WDL 1.0 shield](https://img.shields.io/badge/WDL-1.0-lightgrey.svg)](https://github.com/openwdl/wdl/blob/main/versions/1.0/SPEC.md)  
-This is a work-in-progress project to implement some components of the University of Washington [TOPMed pipeline](https://github.com/UW-GAC/analysis_pipeline) into Workflow Description Language (WDL) in a way that closely mimics [the CWL version of the UW Pipeline](https://github.com/UW-GAC/analysis_pipeline_cwl). In other words, this is a WDL that mimics a CWL that mimics a Python pipeline. All three pipelines use the same underlying R scripts which do most of the heavy lifting, making their results directly comparable.
+This project is a Workflow Description Language (WDL) implementation of several components of the University of Washington [TOPMed pipeline](https://github.com/UW-GAC/analysis_pipeline), purposely done in a way that closely mimics [the CWL version of the UW Pipeline](https://github.com/UW-GAC/analysis_pipeline_cwl). In other words, this is a WDL that mimics a CWL that mimics a Python pipeline. All three pipelines use the same underlying R scripts which do most of the actual analysis, making their results directly comparable.
```

Comment:
Suggest adding as last sentence: "We have also used checker-workflows to verify that results are scientifically equivalent."